src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,411 @@
+- __Create a new request__
 
+POST /v4/requests
 
-## <a name="method1"></a>Retrieve all requests
-						
-	GET  /api/v3.0/travelrequest/requests
-
-### Parameters
+*Parameters:*
+| Name | Parameter Type | Data Type | Description |
+| :---: | :-------: | :--------: | ------ |
+| - | body | [Request](#request) | Required The content of the Request to create |
 
-Name | Type | Format | Description
------|------|--------|------------
-`offset`	|	`string`	|	`query`	|	Starting page offset	
-`limit`	|	`Int32`	|	`query`	|	Number of records to return (default 100)	
-`user`	|	`string`	|	`query`	|	The login ID of the user who owns this Request. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter.	
-`status`	|	`string`	|	`query`	|	The Status search term specifies which travel request or approval status to return. If no Status value is sent, the default Status of Active will be used.	
-`modifiedAfter`	|	`DateTime`	|	`query`	|	This returns travel requests in which the associated dependents (header, entries, segments, allocations, attendees, comments) were modified after the specified date and time. This search term can be used along with other search terms to narrow the results. The date and time (if desired) should be in UTC. The format is: YYYY-MM-DDThh:mm:ss.	
-`modifiedBefore`	|	`DateTime`	|	`query`	|	This returns travel requests in which the associated dependents (header, entries, segments, allocations, attendees, comments) were modified before the specified date and time. This search term can be used along with other search terms to narrow the results. The date and time (if desired) should be in UTC. The format is: YYYY-MM-DDThh:mm:ss.	
-`withSegmentTypes`	|	`Boolean`	|	`query`	|	Pass true to populate the SegmentType field in the result.	
+*Response:*
 
+[Request](#request)
 
-							
-## <a name="method2"></a>Retrieve a request by ID	
+- __Get the list of existing requests__
 
-	GET  /api/v3.0/travelrequest/request/{id}	
+GET /v4/requests
 
-### Parameters
+*Parameters:*
+No parameter
 
+*Response:*
 Name | Type | Format | Description
------|------|--------|------------
-`id`	|	`string`	|	`path`	|	**Required** The Request ID	
-`user`	|	`string`	|	`query`	|	The login ID of the user. Optional. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter.	
-							
-## <a name="method3"></a>Submit a request by ID
-						
-	POST  /api/v3.0/travelrequest/requests/{id}/submit
+--- | :---: | :---: | ------
+data | array | [Request](#request) | List of requests
+operations | string | [RFC 5988](https://tools.ietf.org/html/rfc5988) | Pagination links to next/prev/first/last pages
+
+- __Get the content of an existing request__
+
+GET /v4/requests/{requestUuid}
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | Required The unique identifier of the Request
+
```

Comment:
_note: as per https://rqa3-raas.concurasp.com/#!/request-resource/getListUsingGET_1
additional parameters = userId | query | string | not mandatory_